common/app/views/main.scala.html

```diff
@@ -27,7 +27,7 @@
 }
 
 <!DOCTYPE html>
-<html id="js-context" class="js-off is-not-modern id--signed-out" data-page-path="@request.path">
+<html id="js-context" class="js-off is-not-modern id--signed-out" lang="en" data-page-path="@request.path">
```

Comment:
I think you have won me over with the former, apart from a very select few cases it would be more useful to have the doc language set.